.nrepl-port

```diff
@@ -1 +0,0 @@
-.shadow-cljs/nrepl.port
```

Comment:
This was an intentional link to `.shadow-cljs/nrepl.port`

Reply:
ah sorry, I missed that I will revert, I'll just keep the esclusion in .gitignore since it kept coming up in the untracked files.

Reply:
Sorry, I got my self confused. The reason I removed it is that when I do a `cider-jack-in` into the `clojure-cli`, the contents of that file change to a port number, thus the contents are always replaced with a new port number every time I jack in, and it always appear in git reports as a modified file.

How do you usually start a development environment in Clerk if not jacking in to `clojure-cli`? I'm not familiar with redirecting the nrepl port number to shadow.

Thanks

Reply:
@ikappaki the last part of the README, "Developing Clerk", explains how to start the dev environment using Babashka, after which you would `cider-connect` to the port it prints to your shell.

Reply:
HI @jackrusher, 

thanks! I have never managed to have `bb dev` work for me since I've started looking at dev, other than bringing up the component library page and indulging myself looking at the various UI components available.

The first issue is that he `viewers` version linked from the `:sci` alias in `deps.edn` is pointing to sha that has its `modules/devdocs/deps.edn` pointing to clerk version with a `:git/url` of `git@github.com:nextjournal/clerk.git`. This means that it wont work for anyone other than those who have git access to the site. It has been subsequently updated in a recent `viewers` commit (e06e36dc22ba88ed3852281e5a59de0e3b8c6a2d to be exact). Thus we need to get a new cut of the `viewers` library to make it work. 

```
Cloning: git@github.com:nextjournal/clerk.git
Error building classpath. Unable to clone C:\Users\chaos\.gitlibs\_repos\ssh\github.com\nextjournal\clerk
FATAL ERROR: No supported authentication methods available (server sent: publickey)
fatal: Could not read from remote repository.
```

I meant to be raising an issue for this. I'll do this now. I suppose I can't raise a PR because the particular viewers version to pick up is up to the maintainers to choose.

Then I'm getting a warning about cider and various middlewares not being in sync when i connect to the `bb dev` shadow repl, that put me off so far looking at it (jacking in to clojure-cli directly worked great for me so far). I'll have a look at it next, the `.nrepl-port` sym link comes up in CIDER btw while connecting, but it fails to connect.

Reply:
(raised #40 for the `bb dev` issue)